packages/mobx-state-tree/src/types/utility-types/frozen.ts

```diff
@@ -52,7 +52,7 @@ export function frozen<C>(subType: IType<C, any, any>): IType<C, C, C>
 export function frozen<T>(
     defaultValue: T
 ): IType<T | undefined | null, T, T> & { flags: TypeFlags.Optional }
-export function frozen<T = any>(): IType<T, T, T> // do not assume undefined by default, let the user specify it if needed
+export function frozen<T>(): IType<T, T, T> // do not assume undefined by default, let the user specify it if needed
```

Comment:
breaking-ish change, since create can only be smart if a type is given (although it also works with any), but I think it is better to make the user make an informed decision

Reply:
solved with ts3 in the pull request about ts3